doc/ipad.md

```diff
@@ -50,7 +51,7 @@ refuse to allow WebSockets to connect.
 4. Go to `Settings -> General -> Profile`, select the profile and then hit `Install`.
    - It should say the profile is verified.
 5. Go to `Settings -> About -> Certificate Trust Settings` and enable full trust for
-   the certificate.
+   the certificate. [more apple support here](https://support.apple.com/en-us/HT204477)
```

Comment:
Thanks for adding that link!

Reply:
Absolutely. I got much further with using a Self Assigned Cert but still it ultimately has issues of inconvenience that can be worked around.